src/routes/(app)/product/[id]/+page.server.ts

```diff
@@ -107,6 +108,24 @@ export const load = async ({ params, locals }) => {
 			sort: { percentage: -1 }
 		}
 	);
+	const jsonLd = {
+		'@context': 'https://schema.org/',
+		'@type': 'Product',
+		name: product.name,
+		image: `${ORIGIN}/picture/raw/${pictures[0]._id}/format/${pictures[0].storage.formats[0].width}`,
+		description: product.description,
+
+		brand: {
+			'@type': 'Brand',
+			name: product.name
+		},
+		offers: {
+			'@type': 'AggregateOffer',
```

Comment:
```suggestion
			'@type': 'Offer',
```

`AggregateOffer` is only used when there are multiple offers, and then we need to specify a `highPrice` and `lowPrice`